person.rb

```diff
@@ -0,0 +1,18 @@
+class Person
+  def initialize(age, name = 'Unknown', parent_permission: true)
+    @id = Random.rand(1..1000)
+    @name = name
+    @age = age
+    @parent_permission = parent_permission
+  end
+  attr_accessor :name, :age
+  attr_reader :id
+
+  def of_age?
+    @age >= 18
+  end
```

Comment:
- [ ] Kindly make this method a private method as it is in the project requirements.

![image](https://user-images.githubusercontent.com/86318284/208907670-c87ba84b-7933-430b-a352-84d326c14575.png)

Reply:
Thanks for this. It has been corrected.